src/control/server/init/setup_spdk.sh

```diff
@@ -91,21 +43,13 @@ else
 	${scriptpath}
 	set +x
 
-	# build arglist manually to filter missing directories/files
-	# so we don't error on non-existent entities
 	for glob in '/dev/hugepages' '/dev/uio*'		\
-		'/sys/class/uio/uio*/device/config'	\
+		'/sys/class/uio/uio*/device/config'		\
 		'/sys/class/uio/uio*/device/resource*'; do
 
-		# shellcheck disable=SC2086
-		if list=$(ls -d $glob); then
-			echo -n "RUN: ls -d $glob | xargs -r chown -R "
-			echo "$_TARGET_USER"
-			echo "$list" | xargs -r chown -R "$_TARGET_USER"
+		if compgen -G "${glob}"; then
+			echo "RUN: chown -R "${_TARGET_USER}" "${glob}
```

Comment:
(lint) The surrounding quotes actually unquote this. Remove or escape them. [SC2027]
(lint) Double quote to prevent globbing and word splitting. [SC2086]
(lint) Double quote to prevent globbing and word splitting. [SC2086]